src/main/java/com/webonise/todoapp/controller/UserController.java

```diff
@@ -0,0 +1,57 @@
+package com.webonise.todoapp.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.webonise.todoapp.Exception.InvalidCredentialsException;
+import com.webonise.todoapp.Exception.UserNotExistsException;
+import com.webonise.todoapp.dao.UserRepository;
+import com.webonise.todoapp.model.AuthenticationRequest;
+import com.webonise.todoapp.model.AuthenticationResponse;
+import com.webonise.todoapp.model.UserData;
+import com.webonise.todoapp.service.impl.UserDetailsServiceImpl;
+import com.webonise.todoapp.service.impl.UserServiceImpl;
+import com.webonise.todoapp.util.JwtUtil;
+
+@RestController
+public class UserController {
+
+  @Autowired
+  private AuthenticationManager authenticationManager;
+  @Autowired
+  private JwtUtil jwtUtilToken;
+  @Autowired
+  private UserDetailsServiceImpl userDetailsServiceImpl;
+  @Autowired
+  private UserServiceImpl userServiceImpl;
+  @Autowired
+  private UserRepository userRepository;
+
+  @PostMapping("/signUp")
```

Comment:
Do not use camel casing into the Routs. Use `-` also do not use special characters.